proto/poktroll/application/event.proto

```diff
@@ -46,3 +55,30 @@ message EventTransferError {
   poktroll.application.Application source_application = 3;
   string error = 4;
 }
+
+// EventApplicationUnbondingBegin is emitted when an application unstake message
+// is committed, indicating that an application has begun unbonding.
+message EventApplicationUnbondingBegin {
+  string app_address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
Add `unbonding_height` field for consistency with `EventSupplierUnbondingBegin`.